### ISSUE TITLE
Syntax error causes error

### ISSUE THREAD

**Issue by willplaehn** [kaste/SublimeLinter-contrib-ruff#2]
As of version 0.5.0, Ruff doesn't return an error code of "E999" when syntax errors occur, it returns `null`:

- https://github.com/astral-sh/ruff/blob/main/CHANGELOG.md#050
- https://github.com/astral-sh/ruff/pull/11901

This is causing the following error when a syntax error is encountered:

```python
SublimeLinter.lint.backend [ERROR]: Unhandled exception:
Traceback (most recent call last):
  File "/Users/me/Library/Application Support/Sublime Text/Installed Packages/SublimeLinter.sublime-package/lint/backend.py", line 157, in execute_lint_task
    errors = linter.lint(code, view_has_changed)
  File "/Users/me/Library/Application Support/Sublime Text/Installed Packages/SublimeLinter.sublime-package/lint/linter.py", line 1177, in lint
    return self.filter_errors(self.parse_output(output, virtual_view))
  File "/Users/me/Library/Application Support/Sublime Text/Installed Packages/SublimeLinter.sublime-package/lint/linter.py", line 1201, in filter_errors
    return [
  File "/Users/me/Library/Application Support/Sublime Text/Installed Packages/SublimeLinter.sublime-package/lint/linter.py", line 1201, in <listcomp>
    return [
  File "/Users/me/Library/Application Support/Sublime Text/Installed Packages/SublimeLinter-contrib-ruff.sublime-package/linter.py", line 130, in parse_output
    error_type="error" if code.startswith("F") else "warning",
AttributeError: 'NoneType' object has no attribute 'startswith'
```

**Comment by kaste** [comment]
Checking ... ah I'm on 0.4.5 ... what do they report now?  Everything must have a rule name;  that's actually a nicety because I typically style such hard errors differently, e.g. I literally use a bomb-icon for them right now.  😬 

It's okay and obvious from a user standpoint that you cannot exclude such a syntax-error-rule.  The changelog states "[syntax-error](https://docs.astral.sh/ruff/rules/syntax-error/) (E999): Syntax errors are now always shown"

**Comment by kaste** [comment]
Should be fixed in v1.1.4 -- It takes up to 3 hours before this release gets pushed to the Sublime clients via Package Control.  

Thanks for reporting!
